plugins/vcpu_count/spec.rb

```diff
@@ -0,0 +1,112 @@
+require_relative './plugin.rb'
+describe 'VCPUCount plugin' do
+  before(:all) do
+    @db_table = Iam.DB[:vcpu_count_measurements]
+  end
+
+  # Register method
+  describe '.register method' do
+    it 'does not raise an error when invoked' do
+      expect { VCPUCount.new.register }.to_not raise_error
+    end
+
+    it 'creates a vcpu_count_measurements table' do
+      # Table shouldn't exist before registration
+      expect do
+        Iam.settings.DB.table_exists?(:vcpu_count_measurements).to be_false
+      end
+
+      VCPUCount.new.register
```

Comment:
@ElijahCaine That does make more sense, since everywhere we use the plugin we register it first to make sure the migrations are called. Will fix.